src/Tasks/Microsoft.Common.CrossTargeting.targets

```diff
@@ -46,6 +53,18 @@ Copyright (C) Microsoft Corporation. All rights reserved.
     </ItemGroup>
   </Target>
 
+  <Target Name="GetTargetFrameworksWithPlatformFromInnerBuilds"
+          DependsOnTargets="_ComputeTargetFrameworkItems">
+
+    <MSBuild Projects="@(_InnerBuildProjects)"
+             Condition="'@(_InnerBuildProjects)' != '' "
```

Comment:
nit: `'@(_InnerBuildProjects->Count())' == '0'` could be faster and avoid having spurious whitespace accidentally trigger the condition.